### PR TITLE
Improve networks page

### PR DESCRIPTION
**Pull request by moose-code**
Small things: 

- Brought banner to top explaining tiers
- Made traces checkmark visible
- Capitalized tier names properly

### CODE REVIEW

**Review by JonoPrest — COMMENTED** [review]
@moose-code I think this doc is auto generated/updated from the script? Can you make sure the script includes the updates you want?

**Comment by moose-code** [comment]
> @moose-code I think this doc is auto generated/updated from the script? Can you make sure the script includes the updates you want?

Ahh! Good catch, didn't know that. Thanks I should review again then and change the script 🙏

**Review by JasoonS — CHANGES_REQUESTED** [review]
Yeah, the display isn't affected by this change. Please just check that your change didn't break the auto get (I think it did, because the regex doesn't catch those spaces)

This is the script: https://github.com/enviodev/docs/blob/main/scripts/update-endpoints.js
I think it will match on your change, but the spaces on the other lines won't show. Just verify this (can delete a line such as arbitrum, and it should re-apear after you run `yarn dev`).

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md [hunk 1]
```diff
@@ -70,3 +70,5 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 
 
 
+
+
```
Comment (JasoonS) [inline docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:74]: :thinking:

Comment (JasoonS) [reply docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:74]: Is this because of my script?

Comment (JasoonS) [reply docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:74]: Fixed here: https://github.com/enviodev/docs/pull/404

**Review by JasoonS — APPROVED** [review]
Merging

**Pull request merged**